browser/brave_news/brave_news_controller_factory.h

```diff
@@ -20,7 +23,12 @@ class BraveNewsController;
 
 class BraveNewsControllerFactory : public BrowserContextKeyedServiceFactory {
  public:
-  static BraveNewsController* GetForContext(content::BrowserContext* context);
+  // static BraveNewsController* GetForContext(
+  // content::BrowserContext* context);
```

Comment:
nit: remove

Reply:
@petemill i modified of this method. left the comment for reference. is this ok?

Reply:
This doesn't seem 100% right to me at first glance. If we need access as a remote then maybe an additional static method instead of changing the existing one? Another c++ class should be able to get the BraveNewsController for the context without being wrapped in PendingRemote.